static/routes.config.php

```diff
@@ -638,6 +642,8 @@
 
 	'/update_community[/{content}]'  => [Module\Update\Community::class,      [R::GET]],
 
+	'/update_display[/]'             => [Module\Update\Display::class, [R::GET]],
```

Comment:
Can we just remove the trailing slash in calls and get rid of the conditional part?